llama_hub/llama_packs/tables/mix_self_consistency/mix_self_consistency.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
<p>the paper link is broken </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama-hub/pull/861/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
fixed, also fixed for chain of tables.